_posts/2017-06-01-oitint-swift-dbaland.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "June 1: Meet OIT Integrations, Swift, and Happenings in DBA Land"
+date: 2017-06-01 9:30
+categories: upcoming
+---
+
+- Location: [Walter Library](http://campusmaps.umn.edu/tc/map.php?building=042), Room 402
+- Day: Thursday, June 1st
+- Time: 9:30 - 11:00
+
+Tentative Agenda:
+
+- 9:30 - 9:50 Meet a Dev Group: OIT Integrations - Chris Dinger
```

Comment:
@chris-meyer technically our group is called "Architecture and Systems Integration" or "ASI".